[2025-02-18T01:00:08.767+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T01:00:08.788+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: default_args_dag.task2 scheduled__2025-02-18T00:00:00+00:00 [queued]>
[2025-02-18T01:00:08.798+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: default_args_dag.task2 scheduled__2025-02-18T00:00:00+00:00 [queued]>
[2025-02-18T01:00:08.798+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T01:00:08.813+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): task2> on 2025-02-18 00:00:00+00:00
[2025-02-18T01:00:08.817+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=10572) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-18T01:00:08.818+0000] {standard_task_runner.py:72} INFO - Started process 10573 to run task
[2025-02-18T01:00:08.819+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'default_args_dag', 'task2', 'scheduled__2025-02-18T00:00:00+00:00', '--job-id', '991', '--raw', '--subdir', 'DAGS_FOLDER/12_default_args_dag.py', '--cfg-path', '/tmp/tmpc2190818']
[2025-02-18T01:00:08.820+0000] {standard_task_runner.py:105} INFO - Job 991: Subtask task2
[2025-02-18T01:00:08.873+0000] {task_command.py:467} INFO - Running <TaskInstance: default_args_dag.task2 scheduled__2025-02-18T00:00:00+00:00 [running]> on host ab74d4d90d32
[2025-02-18T01:00:08.988+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='test@test.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='default_args_dag' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2025-02-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-18T00:00:00+00:00'
[2025-02-18T01:00:08.989+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T01:00:09.008+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-18T01:00:09.009+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 5']
[2025-02-18T01:00:09.018+0000] {subprocess.py:99} INFO - Output:
[2025-02-18T01:00:14.022+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-18T01:00:14.051+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T01:00:14.051+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=default_args_dag, task_id=task2, run_id=scheduled__2025-02-18T00:00:00+00:00, execution_date=20250218T000000, start_date=20250218T010008, end_date=20250218T010014
[2025-02-18T01:00:14.097+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T01:00:14.124+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-18T01:00:14.125+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
